pkg/api/deploymentapi/deploymentsize/sizes.go

```diff
@@ -26,14 +26,14 @@ import (
 
 const minsize = 512
 
-// Parse converts the stringified size notation to an int32 Megabyte value.
-// The minimum size allowed is 512 Megabytes.
+// Parse converts the stringified gigabyte size notation to an int32 Megabyte
+// notation. The minimum size allowed is 0.5g Megabytes with 0.5g increments.
 func Parse(strSize string) (int32, error) {
```

Comment:
Since the functionality has changed I think it might be good to change the function's name to `ParseToGb` or something like that. What do you think?

Reply:
I forgot about this PR, I changed the name to `ParseGb` let me know if that's alright! 🙂

Reply:
😄